### ISSUE TITLE
Memory required for UNAGI?

### ISSUE THREAD

**Issue by CWYuan08** [iMetOsaka/UNAGI#11]
Hi, we have some big PromethION data, which we hope to run the UNAGI pipeline with. The cat fastq file is 200G! 
And UNAGI would start with Reading the input file but got killed after a short time. What could memory will this require? 

Many thanks!
best,
CW

**Comment by imet-k** [comment]
Thank you for using our tool. Can you share the error message? to know exactly the step that caused the crash. Also can you share the log related to your command? You can find the log file as /app/log/main.log (last few lines).

Best,

**Comment by CWYuan08** [comment]
Dear @imet-k, 
thanks for checking. I am currently running only a subset of the original file. Only 10k reads now, however the log says: Coverage Breaks + Cuts: Found 248 chromosomes and 61 genes total, which doesn't really make sense. Just to double check for the reference, we only need the genomic reference but no annotation file right? 
And after the run, with the coordinates, what's the recommended downstream pipeline? I would like to do some similar analysis as the paper, is a reference code for this? 
last question: is there an expected percentage for the strandness of the reads. From my last run, it says only 33% were successfully stranded. 
Many thanks for your help!

Best,
CW

**Comment by imet-k** [comment]
Yes. Number of chromosome depends on the provided reference. Low expressed genes are excluded and this is probably why the gene number was low since you had only 10k reads input. 
Stranding percentage should be more. It depends on the protocol to prepare the library. What was your protocol or kit used to prepare the cDNA?

Best,
